app-modules/contact/src/Providers/ContactServiceProvider.php

```diff
@@ -65,9 +71,12 @@ public function boot(): void
             'contact' => Contact::class,
             'contact_source' => ContactSource::class,
             'contact_status' => ContactStatus::class,
+            'organization' => Organization::class,
```

Comment:
We should add the `OrganizationType` and `OrganizationIndustry` models here as well.

Reply:
Done